src/site/content/en/blog/storage-for-the-web/index.md

```diff
@@ -96,9 +94,14 @@ device.
 * Chrome allows the browser to use up to 80% of total disk space. An origin can
   use up to 60% of the total disk space. You can use the [StorageManager
   API](#check) to determine the maximum quota available. Other Chromium-based
-  browsers may allow the browser to use more storage. See
-  [PR #3896](https://github.com/GoogleChrome/web.dev/pull/3896) for details about
-  Chrome's implementation.
+  browsers may allow the browser to use more storage.
```

Comment:
Browsers may allow the browser? If this is what you meant, I'm not sure I understand.

Reply:
`Other Chromium-based browsers may allow the browser to use more storage.`

When read as the whole sentence, I think it makes sense, but I'll update it.